packages/flutter/lib/src/material/outlined_button.dart

```diff
@@ -0,0 +1,362 @@
+      padding: scaledPadding,
+      minimumSize: const Size(64, 36),
+      side: BorderSide(
+        color: Theme.of(context).colorScheme.onSurface.withOpacity(0.12),
+        width: 1,
+      ),
+      shape: const RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(4))),
+      enabledMouseCursor: SystemMouseCursors.click,
+      disabledMouseCursor: SystemMouseCursors.forbidden,
+      visualDensity: theme.visualDensity,
+      tapTargetSize: theme.materialTapTargetSize,
+      animationDuration: kThemeChangeDuration,
+      enableFeedback: true,
+    );
+  }
+
+  @override
+  ButtonStyle themeStyleOf(BuildContext context) {
+    return OutlinedButtonTheme.of(context)?.style;
+  }
+}
+
+@immutable
+class _OutlinedButtonDefaultForeground extends MaterialStateProperty<Color>  with Diagnosticable {
+  _OutlinedButtonDefaultForeground(this.primary, this.onSurface);
+
+  final Color primary;
+  final Color onSurface;
+
+  @override
+  Color resolve(Set<MaterialState> states) {
+    if (states.contains(MaterialState.disabled))
+      return onSurface?.withOpacity(0.38);
+    return primary;
+  }
+}
+
+@immutable
+class _OutlinedButtonDefaultOverlay extends MaterialStateProperty<Color> with Diagnosticable {
+  _OutlinedButtonDefaultOverlay(this.primary);
+
+  final Color primary;
+
+  @override
+  Color resolve(Set<MaterialState> states) {
+    if (states.contains(MaterialState.hovered))
+      return primary?.withOpacity(0.04);
+    if (states.contains(MaterialState.focused) || states.contains(MaterialState.pressed))
+      return primary?.withOpacity(0.12);
+    return null;
+  }
+}
+
+@immutable
+class _OutlinedButtonDefaultMouseCursor extends MaterialStateProperty<MouseCursor> with Diagnosticable {
+  _OutlinedButtonDefaultMouseCursor(this.enabledCursor, this.disabledCursor);
+
+  final MouseCursor enabledCursor;
+  final MouseCursor disabledCursor;
+
+  @override
+  MouseCursor resolve(Set<MaterialState> states) {
+    if (states.contains(MaterialState.disabled))
+      return disabledCursor;
+    return enabledCursor;
+  }
+}
+
+class _OutlinedButtonWithIcon extends OutlinedButton {
+  _OutlinedButtonWithIcon({
+    Key key,
+    @required VoidCallback onPressed,
+    VoidCallback onLongPress,
+    ButtonStyle style,
+    FocusNode focusNode,
+    bool autofocus,
+    Clip clipBehavior,
+    @required Widget icon,
+    @required Widget label,
+  }) : assert(icon != null),
+       assert(label != null),
+       super(
+         key: key,
+         onPressed: onPressed,
+         onLongPress: onLongPress,
+         style: style,
+         focusNode: focusNode,
+         autofocus: autofocus ?? false,
+         clipBehavior: clipBehavior ?? Clip.none,
+         child: _OutlinedButtonWithIconChild(icon: icon, label: label),
+      );
+
+  @override
+  ButtonStyle defaultStyleOf(BuildContext context) {
+    final EdgeInsetsGeometry scaledPadding = ButtonStyleButton.scaledPadding(
+      const EdgeInsets.all(8),
```

Comment:
This should use the same padding as OutlinedButtons with Icons, (12 leading and 16 trailing)

Reply:
Done.